docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -1717,8 +1749,890 @@ Here are the steps:
+
+- Domain in Image requires that its images embed a WebLogic `security/SerializedSystemIni.dat` domain encryption key that cannot be changed for the image (see [why layering matters]({{< relref "/userguide/cicd/why-layering-matters.md" >}}) in CI/CD Considerations). This necessarily means that two Domain in Image domains that share the same image have the ability to decrypt each other's encrypted passwords. A Model in Image's domain encryption key, on the other hand, is not embedded in the image and is instead dynamically and uniquely created each time the domain is started. 
+
+{{% notice warning %}}
+Oracle requires interoperating WebLogic domains to have different domain names. This is necessary when two domains communicate, or when a WebLogic Server or WebLogic java client concurrently connects to multiple domains. 
+{{% /notice %}}
+
+Here are the steps for this use case:
+
+1. Make sure you have deployed the domain from the [Update1 use case](#update1-use-case).
+
+1. Create a ConfigMap with the WDT model that contains the data source definition.
+
+   Run the following commands:
+
+   ```
+   $ kubectl -n sample-domain1-ns create configmap sample-domain2-wdt-config-map \
+     --from-file=/tmp/mii-sample/model-configmaps/datasource
+   $ kubectl -n sample-domain1-ns label configmap sample-domain2-wdt-config-map \
+     weblogic.domainUID=sample-domain2
+   ```
+
+   If you've created your own data source file in the Update1 use case, then substitute the file name in the `--from-file=` parameter (we suggested `/tmp/mii-sample/mydatasource.yaml` earlier). Note that the `-from-file=` parameter can reference a single file, in which case it puts the designated file in the ConfigMap, or it can reference a directory, in which case it populates the ConfigMap with all of the files in the designated directory.
+
+   Observations:
+     - We are leaving the namespace `sample-domain1-ns` unchanged for the ConfigMap because we plan to deploy domain `sample-domain2` to the same namespace as `sample-domain1`.
+     - We name and label the ConfigMap using its associated domain UID for two reasons:
+       - To make it obvious which ConfigMap belongs to which domain.
+       - To make it easier to cleanup a domain. Typical cleanup scripts use the `weblogic.domainUID` label as a convenience for finding all resources associated with a domain.
+     - We use a different ConfigMap for the new domain for two reasons:
+       - To make it easier to keep the life-cycle and/or CI/CD process for the two domains simple and independent. Any shared resources between the two domains must necessarily have their own life-cycle.
+       - To 'future proof' the new domain so that changes to the original domain or new domain's ConfigMap can be independent.
+
+
+1. Create the secrets that are referenced by the WDT model files in the image and ConfigMap, and that will be referenced by the domain resource.
```

Comment:
and that will be referenced -> and they will be referenced

Reply:
dispute this one

Reply:
dispute resolved - compromise change made